Resources/Prototypes/Entities/Objects/Specific/Medical/defib.yml

```
@@ -66,7 +66,7 @@
   suffix: One-Handed, Always Powered
 
 - type: entity
-  id: DefibrillatorCompact # This should be a research item at some point
+  id: DefibrillatorCompact
```

```suggestion
  id: DefibrillatorCompact # This should be a research item at some point
```
lets keep the upstream comments untouched, unnecessary